### PR TITLE
Fix for issue #13531

### PR DESCRIPTION
**Pull request by frankmayer**
Pull Request for Issue #13531 .

### Summary of Changes
- [AND] and [OR] operators were not functioning correctly. Fixed.
- some cleanup of whitespaces in XML file and hiding of rulers when the encapsulated fields are not showing.

### Testing Instructions
Code review (with w=1 because of the whitespace reformat in the xml file) [here](https://github.com/joomla/joomla-cms/pull/13535/files?w=1
)

Test fields where showon with [AND] and [OR] are implemented.
for example:
[AND]: Articles->Options->"Articles" tab
Only when both "Show "Read More" and "Show Title with Read More" are set to "Show" the "Read More Limit" field should be shown.

[OR]: Articles->Options->"Editing Layout" tab
Only when both "Frontend Images and Links" and "Administrator Images and Links" are set to "Hide" then the five fields under them should be hidden.

### Documentation Changes Required
None

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on fe73679c58e69fd7d3ddd66aa549c0c76959fc64<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13535">issues.joomla.org/tracker/joomla-cms/13535</a>.</sub>

**Comment by andrepereiradasilva** [comment]
conflicts, also it seems you change the config.xml indentation. please revert that extra indentation

**Comment by zero-24** [comment]
@frankmayer please reduce the change in the config.xml only to the needed one as the tabs you added are not needed. as @andrepereiradasilva pointet out. that should also fix the merge conflicts.

File: administrator/components/com_content/config.xml [hunk 1]
```diff
@@ -516,47 +518,47 @@
...
-			description="JGLOBAL_SHOW_CATEGORY_TITLE_DESC"
-			default="1"
-			>
-				<option value="1">JSHOW</option>
-				<option value="0">JHIDE</option>
+			   type="radio"
+			   class="btn-group btn-group-yesno"
+			   label="JGLOBAL_SHOW_CATEGORY_TITLE"
+			   description="JGLOBAL_SHOW_CATEGORY_TITLE_DESC"
+			   default="1"
```
Comment (zero-24) [inline administrator/components/com_content/config.xml]: hmm this changes still looks wrong to me (spaces and no tabs ;))

Comment (frankmayer) [inline administrator/components/com_content/config.xml]: It's a setting to align the attributes... but I know, not everyone uses the same tab width...

**Comment by frankmayer** [comment]
@zero-24 & @andrepereiradasilva it was a different than Joomla style, code formatting setting for XML in PHPStorm.

**Comment by frankmayer** [comment]
Conflicts and whitespace issues have been resolved. Tests and RTC anyone?

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 62e6fc1e4209ca5049d4df4e5626b97142012c9b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13535">issues.joomla.org/tracker/joomla-cms/13535</a>.</sub>

**Comment by anibalsanchez** [comment]
I have tested this item :white_check_mark: successfully on 62e6fc1e4209ca5049d4df4e5626b97142012c9b<br /><br />Test OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13535">issues.joomla.org/tracker/joomla-cms/13535</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13535">issues.joomla.org/tracker/joomla-cms/13535</a>.</sub>

**Comment by wilsonge** [comment]
There are now conflicts after I merged https://github.com/joomla/joomla-cms/pull/13549 - these need to be fixed and we need one more test once you've fixed the conflicts

**Comment by frankmayer** [comment]
Fixing a bug in conflict resolution... PLS don't test yet

**Comment by frankmayer** [comment]
Conflicts resolved... pls test and RTC

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 086ee73ea3927152da5fe14aceadff54f03dc050<br /><br />The [AND] and the [OR] example works.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13535">issues.joomla.org/tracker/joomla-cms/13535</a>.</sub>

**Pull request merged**